config/clients/python/template/api_test.mustache

```diff
@@ -0,0 +1,1202 @@
+                    object="document:2021-budget",
+                    relation="reader",
+                    user="anne",
+                ),
+            )
+            api_response = api_instance.check(
+                body=body,
+            )
+            self.assertIsInstance(api_response, CheckResponse)
+            self.assertTrue(api_response.allowed)
+            # Make sure the API was called with the right data
+            mock_request.assert_called_once_with(
+                'POST',
+                'http://api.{{sampleApiDomain}}/stores/d12345abc/check',
+                headers=ANY,
+                query_params=[],
+                post_params=[],
+                body={"tuple_key":{"object":"document:2021-budget","relation":"reader","user":"anne"}},
+                _preload_content=ANY,
+                _request_timeout=None
+            )
+
+    @patch.object({{packageName}}.api_client.ApiClient, 'request')
+    def test_create_store(self, mock_request):
+        """Test case for create_store
+
+        Create a store  # noqa: E501
+        """
+        response_body = '''{
+            "id": "01YCP46JKYM8FJCQ37NMBYHE5X",
+            "name": "test_store",
+            "created_at": "2022-07-25T17:41:26.607Z",
+            "updated_at": "2022-07-25T17:41:26.607Z"}
+            '''
+        mock_request.return_value = mock_response(response_body, 201)
+
+        configuration = self.configuration
+        with {{packageName}}.ApiClient(configuration) as api_client:
+            api_instance = open_fga_api.OpenFgaApi(api_client)
+            body = CreateStoreRequest(
+                name="test-store",
+            )
+            api_response = api_instance.create_store(
+                body=body,
+            )
+            self.assertIsInstance(api_response, CreateStoreResponse)
+            self.assertEqual(api_response.id, '01YCP46JKYM8FJCQ37NMBYHE5X')
+            mock_request.assert_called_once_with(
+                'POST',
+                'http://api.{{sampleApiDomain}}/stores',
+                headers=ANY,
+                query_params=[],
+                post_params=[],
+                body={"name":"test-store"},
+                _preload_content=ANY,
+                _request_timeout=None
+            )
+
+    @patch.object({{packageName}}.api_client.ApiClient, 'request')
+    def test_delete_store(self, mock_request):
+        """Test case for delete_store
+
+        Delete a store  # noqa: E501
+        """
+        response_body = ''
+        mock_request.return_value = mock_response(response_body, 201)
+        configuration = self.configuration
+        configuration.store_id = store_id
+        with {{packageName}}.ApiClient(configuration) as api_client:
+            api_instance = open_fga_api.OpenFgaApi(api_client)
+            api_instance.delete_store()
+            mock_request.assert_called_once_with(
+                'DELETE',
+                'http://api.{{sampleApiDomain}}/stores/d12345abc',
```

Comment:
why is store id hard coded here?

Reply:
It is test and we are writing hard coding store id in line 58.